mmv1/third_party/terraform/tests/resource_cloud_run_v2_service_test.go

```diff
@@ -407,3 +434,107 @@ resource "google_cloud_run_v2_service" "default" {
 }
 `, context)
 }
+
+
+func testAccCloudRunV2Service_cloudRunServiceUpdateWithEmptyGRPCLivenessProbe(context map[string]interface{}) string {
+	return Nprintf(`
+resource "google_cloud_run_v2_service" "default" {
+  name     = "tf-test-cloudrun-service%{random_suffix}"
+  location = "us-central1"
+
+  metadata {
+    namespace = "%s"
+    annotations = {
+      generated-by = "magic-modules"
+    }
+  }
+
+  template {
+    containers {
+      image = "gcr.io/cloudrun/hello"
```

Comment:
Google Cloud does not provide any gRPC variant of official container images for Cloud Run Services. `gcr.io/cloudrun/hello` is only for REST API and Cloud Event processing servers.

To make this E2E tests more realistic, it would be desirable to use a container image of a real gRPC server that implements the gRPC health check service.
So I [created](https://github.com/mmizutani/cloud-run-hello-grpc) such a custom docker image, but it would be more desirable to use more "official" container images hosted by Google Cloud or Hashicorp.